[2024-01-11T18:21:28.738+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop manual__2024-01-11T18:16:02+00:00 [queued]>
[2024-01-11T18:21:28.742+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop manual__2024-01-11T18:16:02+00:00 [queued]>
[2024-01-11T18:21:28.743+0000] {taskinstance.py:2171} INFO - Starting attempt 2 of 2
[2024-01-11T18:21:28.749+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): run_after_loop> on 2024-01-11 18:16:02+00:00
[2024-01-11T18:21:28.754+0000] {standard_task_runner.py:60} INFO - Started process 298 to run task
[2024-01-11T18:21:28.758+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2024-01-11T18:16:02+00:00', '--job-id', '15', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp48qxtixa']
[2024-01-11T18:21:28.762+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask run_after_loop
[2024-01-11T18:21:28.801+0000] {task_command.py:423} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2024-01-11T18:16:02+00:00 [running]> on host ca44f6985e16
[2024-01-11T18:21:28.853+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='run_after_loop' AIRFLOW_CTX_EXECUTION_DATE='2024-01-11T18:16:02+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-11T18:16:02+00:00'
[2024-01-11T18:21:28.855+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-11T18:21:28.855+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 1']
[2024-01-11T18:21:28.867+0000] {subprocess.py:86} INFO - Output:
[2024-01-11T18:21:28.869+0000] {subprocess.py:93} INFO - 1
[2024-01-11T18:21:28.870+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-11T18:21:28.888+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20240111T181602, start_date=20240111T182128, end_date=20240111T182128
[2024-01-11T18:21:28.930+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-11T18:21:28.942+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
